android/app/build.gradle

```diff
@@ -26,7 +26,7 @@ apply plugin: 'kotlin-android'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
-    compileSdkVersion 29
+    compileSdkVersion 31
```

Comment:
is there a reason to bump this?

Reply:
the new version of flutter was complaining about the version of the gradle plugins so I bumped them and then I had to bump this because some dependencies use compileSdkVersion 31